core/shared/src/main/scala/org/http4s/util/UrlCoding.scala

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Darn, somehow git still couldn't track this file.

Reply:
That's fine.